### ISSUE TITLE
Merge strand-bias branch into master

### ISSUE THREAD

**Issue by ArtPoon** [cfe-lab/MiCall#257]
Make this part of version 7.1

**Comment by donkirkby** [comment]
The branch never got merged in, @ArtPoon. Should we do that? Is there any documentation for what it does? Any suggestions on how to add tests for the new features?

**Comment by ArtPoon** [comment]
The reason I made this branch was to extract strand-bias statistics so that I could generate reports containing metrics required by Merck.  I wrote it in a hurry and there's still some debugging code in there, e.g., lines 112-115 in `micall/core/aln2counts.py`.

I don't know whether the lab is still interested in having these metrics.  My gut feeling is to discard the branch to minimize disruption to the lab pipeline.
